File: 2D-World-Simulator-CPP/OPP-CPP/Grass.cpp
#include "Grass.h"

Grass::Grass(World* world, Point point) : Plant(world, point) {

	this->setOrganismSign('#');
	this->setStrength(0);
	this->setInitiative(0);
	this->setOrganismSpecie(OrganismSpecie::GRASS);

}

File: 2D-World-Simulator-Java/OPP-Java/src/op/JavaProject/Organisms/Plants/Grass.java
package op.JavaProject.Organisms.Plants;

import op.JavaProject.Functionality.BoardPoint;
import op.JavaProject.Functionality.Species;
import op.JavaProject.Game.Plant;
import op.JavaProject.Game.World;

import java.awt.*;

public class Grass extends Plant {

    public Grass(World organismWorld, BoardPoint point) {
        super(organismWorld, point);
        this.strength = 0;
        this.initiative = 0;
        this.organismSign = "#";
        this.organismSpecie = Species.GRASS;
        this.organismColor = new Color(27, 90, 0);
    }

}
